netmiko/nokia/nokia_sros.py

```diff
@@ -53,9 +53,7 @@ def session_preparation(self) -> None:
         else:
             # Classical CLI has no method to set the terminal width nor to disable command
             # complete on space; consequently, cmd_verify needs disabled.
-            # Only disabled if not set under the ConnectHandler.
-            if self.global_cmd_verify is None:
-                self.global_cmd_verify = False
+            self.global_cmd_verify = False
```

Comment:
Looks like a rebasing issue (i.e. undoing a change that was previously merged into the develop branch.

Reply:
Probably.